parm/config/gfs/config.resources

```diff
@@ -1013,13 +1014,14 @@ case ${step} in
     ;;
 
   "esfc")
-    export wtime_esfc="00:08:00"
+    export wtime_esfc="00:15:00"
     export npe_esfc=80
     export nth_esfc=1
     export npe_node_esfc=$(( npe_node_max / nth_esfc ))
     export nth_cycle=${nth_esfc}
     export npe_node_cycle=$(( npe_node_max / nth_cycle ))
     export memory_esfc="80GB"
+    if [[ ${machine} == "JET" ]]; then export memory_esfc="0" ; fi
```

Comment:
What happens if it get "some" memory?
Would it be better to cast it as:
```
if [[ ${machine} != "JET" ]]; then export memory_esfc="80GB" ; fi
```

Reply:
> What happens if it get "some" memory? Would it be better to cast it as:
> 
> ```
> if [[ ${machine} != "JET" ]]; then export memory_esfc="80GB" ; fi
> ```

That will not work on xjet partition; Xjet has a max of 64G/node.

Reply:
That would specifically be if it wasn't on Jet. On Jet the value would be unset.

Reply:
My bad, I read incorrectly. I get it now.

Reply:
@aerorahul, I changed to your suggestion, setting `memory_esfc` only if not on JET.